docs/third-party-developers/extensibility/data-store/collections.md

```diff
@@ -11,6 +11,10 @@
 -   [Selectors](#selectors)
     -   [getCollection](#getcollection)
     -   [getCollectionHeader](#getcollectionheader)
+    -   [getFromState](#getfromstate)
+    -   [getCollectionHeaders](#getcollectionheaders)
+    -   [getCollectionError](#getcollectionerror)
+    -   [getCollectionLastModified](#getcollectionlastmodified)
```

Comment:
What are your thoughts on rearranging the sections in this way? 

```suggestion
    -   [getFromState](#getfromstate)
    -   [getCollection](#getcollection)
    -   [getCollectionHeader](#getcollectionheader)
    -   [getCollectionHeaders](#getcollectionheaders)
    -   [getCollectionError](#getcollectionerror)
    -   [getCollectionLastModified](#getcollectionlastmodified)
```

Please note when doing so, the sections below need to be arranged, otherwise the next time we create the TOC automatically a manual adjustment of the TOC sections would be overwritten.